client/src/components/PlanDetails.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import Modal from 'react-awesome-modal';
+
+class PlanDetails extends React.Component {
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      show: false
+    }
+  }
+
+  openModal() {
+    this.setState({show: true});
+  }
+
+  closeModal() {
+    this.setState({show: false});
+  }
+
+  render() {
+
+    var checkBoxStyle = {
+      height: "15px",
+      width: "15px",
+      cursor: "pointer"
+    }
```

Comment:
;
are you using a linter?

also, I think it would be good to be consistent w/ your use of `"` vs `'` in your js files